### ISSUE TITLE
Next 15 support & peerDependencies bump

### ISSUE THREAD

**Issue by tianyingchun** [expatfile/next-runtime-env#157]
Next 15 support & peerDependencies bump

**Comment by mcorbelli** [comment]
news?

**Comment by Verteletsky** [comment]
any news?

**Comment by AlimovSV** [comment]
any news?

**Comment by anders2410** [comment]
Have anyone tried upgrading to Nextjs 15 to see if it is working? I'm particularly interested in seeing if it works with `next-runtime-env@1.*` and `nextjs@15` with the pages router.

**Comment by AlimovSV** [comment]
Yes, it works for me, I just had to override the nextjs module resolution in package.json file
```json
  "resolutions": {
    "next": "15.1.2"
  }
```

**Comment by HofmannZ** [comment]
We’re actively working on this in the following PR: https://github.com/expatfile/next-runtime-env/pull/164

One challenge we’ve run into is that the `unstable_noStore` API has been removed in Next.js 15. We’ve relied on it to ensure caching is disabled when accessing environment variables.

We’re currently exploring the new `connection` API introduced in v15 as a potential alternative. Feel free to take a look and share any early feedback!

**Comment by itaru2622** [comment]
hi @HofmannZ
I want to try next15 support feature #164 but it was based on v3.3.0-alpha.1 
could you rebase #164 on latest v3.3.0 or develop branch? when I try rebase by myself, it has conflicts and needs to resolve...
best regards.

**Comment by kaleb110** [comment]
how is it working for me in next ^15.3.1 although it doesn't work in next 15.3.5
